modules/default/calendar/calendar.js

```diff
@@ -337,6 +337,9 @@ Module.register("calendar", {
 							}
 							break;
 						}
+						if (typeof this.config.customEvents[ev].eventClass !== "undefined" && this.config.customEvents[ev].eventClass !== "") {
+							eventWrapper.className += ` ${this.config.customEvents[ev].eventClass}`;
```

Comment:
Are you interested in writing a test for this? sjould be in there: https://github.com/MichMich/MagicMirror/blob/abe5c08a52ffee6b37c30c1f3160e130f0e966a2/tests/e2e/modules/calendar_spec.js#L45

Reply:
Done ;-)